pkg/controller/user-cluster-controller-manager/resources/resources/operating-system-manager/secret.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operatingsystemmanager
+
+import (
+	"k8c.io/kubermatic/v2/pkg/resources"
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	cloudInitGetterToken      = "cloud-init-getter-token"
+	cloudInitGetterAnnotation = "cloud-init-getter-token"
```

Comment:
```suggestion
	cloudInitGetterAnnotation = "cloud-init-getter"
```

The annotation should point to the service account name i.e. `cloud-init-getter`

Also, can we move this to https://github.com/kubermatic/kubermatic/tree/master/pkg/controller/user-cluster-controller-manager/resources/resources/cloudinitsettings and create this unconditionally? Since this is also consumed for the bare-metal nodes:

- https://github.com/kubermatic/machine-controller/blob/master/pkg/cloudprovider/util/cloud_init_settings.go#L41
- https://github.com/kubermatic/machine-controller/blob/master/pkg/cloudprovider/provider/baremetal/provider.go#L226